Samples/All/BumbleRumble/Common/NetworkManager.cpp

```diff
@@ -1010,7 +1011,7 @@ void NetworkManager::DoWork()
                 // Toast the text on the screen
                 Managers::Get<ScreenManager>()->GetSTTWindow()->AddSTTString(
                     DisplayNameFromChatControl(result->senderChatControl),
-                    result->transcription,
+                    result->translationCount > 0 && result->translations[0].result == PartyStateChangeResult::Succeeded ? result->translations[0].translation : result->transcription,

```

Comment:
I whipped this out this one-liner to quickly unblock the demo. To improve diagnostics of those using the sample, consider restructuring this a bit so we can log errors when the translation result indicates failure, similar to what's done for the untranslated source text.

Reply:
I should have known you wouldn't let me get away with it :)